chapter1_welcome/chapter1_welcome.md

```diff
@@ -0,0 +1,153 @@
+# Chapter 1 Welcome Your Journey Starts Here
+
+Welcome to the Topcon Sitelink3D v2 API example repository documentation. The chapters contained in this repository make reference to the Python scripts found in our Sitelink3D v2 Examples [here](https://github.com/Sitelink3D-v2-Developer/sitelink3dv2-examples). 
```

Comment:
I think the user should focus on reading this paragraph first before deciding to click onto the link so they're not distracted. We could move the link down to the example code environment subheading.